portfolio/src/main/webapp/style.css

```diff
@@ -4,8 +4,12 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#welcome-message-container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 500px;
+  font-size: 26px;
+  text-align: center;
```

Comment:
nit: CSS declarations should be alphabetized.

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Done!